backend/trojan_dining/views.py

```diff
@@ -81,7 +83,34 @@ def post(self, request):
                 return HttpResponse(400)
             
             try:  
-                Subscription(item_id = item_id, email = email, phone_no = phone_number).save()
+                try:
+                    Subscription.objects.get(item_id = item_id, email = email, phone_no = phone_number)
+                except:
+                    Subscription(item_id = item_id, email = email, phone_no = phone_number).save()
```

Comment:
Is there a way we can do this without a try/except?

Reply:
The alternative would be using the filter method instead of the get method on the Subscription model and getting the first element from that query set. The reason I opted for get is because I never expected for there to be more than 1 instance of a subscription. There isn't really any other way to check if a get successfully found an record other than try except. I can do the filter method instead. I just thought get made sense for this use case.

Reply:
huh, i thought we could just call `.first()` or something. This is fine, thanks for the info 👍